docs/superpowers/specs/2026-04-15-plugin-sidekick-design.md

```
@@ -0,0 +1,102 @@
+
+> **Source of truth:** `packages/plugins/plugin-sidekick/PLUGIN.mdl`
+>
+> This file contains implementation context that supplements the MDL spec:
+> the instruction template text and the operations mapping table.
+
+## Instruction Template
+
+The agent's `spec` document contains a `Template.make()` with the following text.
+Template inputs: `userName` (from identity profile), space context objects.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)